frontend/src/routes/(authenticated)/project/[project_code]/AddProjectMember.svelte

```diff
@@ -23,15 +23,21 @@
     const { response, formState } = await formModal.open(async () => {
       const { error } = await _addProjectMember({
         projectId,
-        userEmail: $form.email,
+        usernameOrEmail: $form.usernameOrEmail,
         role: $form.role,
       });
 
+      if (error?.byType('InvalidOperationError')) {
+        return { usernameOrEmail: [$t('project_page.add_user.project_not_found')] };
```

Comment:
This should put it at the bottom of the form.
```suggestion
        return $t('project_page.add_user.project_not_found');
```